solstdlib/src/precompiles.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Shouldn't these be in a directory called `std` if you `import "std/precompiles.sol"`?